store-query.js

```diff
@@ -1,12 +1,12 @@
 'use strict'
 
 var _ = require('lodash')
-var QueryBuilder = require('./lib/query-builder')
-
 var actionRole = 'sql'
```

Comment:
I know it is not part of the PR, but the name is super-generic. I'm curios how it is used? It is used only internally?